reference-implementation/run-web-platform-tests.js

```diff
@@ -21,11 +21,11 @@ wptRunner(toUpstreamTestsPath, { rootURL: 'streams/', setup })
   })
   .then(failures => {
     totalFailures += failures;
-    process.exit(totalFailures);
+    process.exitCode = totalFailures;
```

Comment:
Wait, why not? Why would the process need to exit immediately, when it's about to exit on its own naturally regardless as this is the last thing that gets run and nothing else is keeping process alive? This seems cleaner.